app/views/request-psp-test-account/index.njk

```diff
@@ -0,0 +1,73 @@
+{% extends "../layout.njk" %}
+
+{% block pageTitle %}
+  Request Stripe test account - {{ currentService.name }} - GOV.UK Pay
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{
+    govukBackLink({
+      text: "My services",
+      href: routes.serviceSwitcher.index
+    })
+  }}
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-two-thirds">
+
+  {% if isServiceLive %}
+    <h1 class="govuk-heading-l">
+      Test account cannot be requested
+    <h1>
+    <p class="govuk-body">
+      Stripe test account cannot be requested for live service. You can create another service and request Stripe test account
```

Comment:
I assume the content here is not finalised? A number of grammatical issues I think 
```suggestion
      Stripe test accounts cannot be requested for a live service. You can create another service and request a Stripe test account
```